sklearn/utils/seq_dataset.pxd.tp

```diff
@@ -0,0 +1,69 @@
+"""Dataset abstractions for sequential data access. """
```

Comment:
It would be useful here to point to what kind of file this is (a Cython template), given an URL where it is described. Indeed, such files are uncommon, and people might be surprised.